### DISCUSSION TITLE
Is there a way to directly use "newer" stubs?

### DISCUSSION THREAD

**Discussion by CaptainCoward** [elishacloud/dxwrapper#148] [General]
For example d3d8to9 just received an update. However using that d3d8.dll instead of the dll in the dxwrapper package doesn't really work (settings will be ignored).

**Comment by elishacloud** [comment]
No, there is no way to use newer builds unless you can recompile the code.  I know there is a new release of d3d8to9, but the code has not changed since Dec 30, 2020, so the latest build of dxwrapper should already have them all.

**Reply by CaptainCoward** [reply]
Ty, good to know :)
